### PR TITLE
fix: Add helper information on page

### LINKED ISSUE #6062: Virtual Events Video Tab: Add helper information on page
Please add helper information on the video tab page to guide users as follows. On the table heading and on the video edit page:
* Microlocations add an (i) icon. When the user scrolls over it show the info "You need to add rooms into the wizard first before they show up here."
* Video Source URL add an (i) icon. When the user scrolls over it show the info "This column shows the original link of the video. We do not recommend to share this link as users can access it without loggin into eventyay.com."
* "Join Video" add an (i) icon. When the user scrolls over it show the info "You can share the links in this columns with attendees of an event. On event or schedule schedule pages attendees of your event will see a Join Video button. These buttons will direct to the links shown here. Only organizers and registered ticket holders will be able to access these links."
* "Room Password" add an (i) icon. When the user scrolls over it show the info "The room password field can be used to communicate a password which is necessary to access online video rooms for example for external video services such a Zoom, Teams and Webex. The need for a password entry depends on the configuration of your video channel. The integrated Big Blue Button video solution in eventyay.com does not need a password as only ticket holders are able to access it. The difference between the password and PIN is that the password option is used for online access while PINs are used to access video rooms through the telephone."
* "Additional Information" add an (i) icon. When the user scrolls over it show the info "Additional Information fields can be used to share information such as phone access, PINs and other video room specific information."

### PR DESCRIPTION
**Pull request by daretobedifferent18**
Fixes #6062 
Video Columns | Video Edit Page
---- | ------------
Screenshot from 2020-12-21 08-55-50 | Screenshot from 2020-12-21 08-55-40

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by daretobedifferent18** [comment]
@iamareebjamal Pls see the changes made

**Comment by nlok5923** [comment]
@daretobedifferent18 there already a pr #6067 resolving this issue.

**Comment by daretobedifferent18** [comment]
> @daretobedifferent18 there already a pr #6067 resolving this issue.

But the PR was not active and also I have not copied ur solution. My solution is diff from you

**Comment by daretobedifferent18** [comment]
@iamareebjamal Pls have a look and let me know to make any changes

**Comment by daretobedifferent18** [comment]
@iamareebjamal Pls have a look and let me know to make any changes

**Comment by mariobehling** [comment]
It creates more work for everyone if people resolve the same issue without collaborating. It is also possible to make a PR to a fork to improve existing work or a contributor. I posted the below feedback already on the other PR. It is also valid on this PR. Please collaborate. Do not blame one another for taking up this or that PR. It is possible that overlap happens, but just resolve it with each other in a positive way please. In any case we need the issue to be resolved asap. Thanks.

> Thank you. Could you change the following please:
> * Please take out the effect of the tool tip and make it appear right away without any delay
> * The way the icon appears currently stands out a lot. It looks larger and bold compared to the text. Therefore please make the icon to the same font size as the text and change the i-icon to an icon more similar to the following. Do not make it appear bolder than the text in front.
Screenshot from 2020-12-18 11-19-47

**Comment by mariobehling** [comment]
So, is it possible to get rid of the expansion effect and make the tool tip visible without delay?

**Comment by daretobedifferent18** [comment]
@mariobehling the icon u mentioned above is not available in simantic ui icons list so either we can use  
Screenshot from 2020-12-23 00-19-53

**Comment by daretobedifferent18** [comment]
> So, is it possible to get rid of the expansion effect and make the tool tip visible without delay?

what I get is that u r asking me to remove the duration for transition/animation effect so that the popup appear intantly?

**Comment by daretobedifferent18** [comment]
> > So, is it possible to get rid of the expansion effect and make the tool tip visible without delay?
> 
> what I get is that u r asking me to remove the duration for transition/animation effect so that the popup appear intantly?

If thats the case then I think we can't manipulate with css or add our own class in UiPopup

**Comment by maze-runnar** [comment]
> So, is it possible to get rid of the expansion effect and make the tool tip visible without delay?

I think there should be a separate issue for this. It would affect a lot of areas of application. And also this PR looks completed to me.

File: app/components/forms/events/view/videoroom-form.hbs [hunk 1]
```diff
@@ -42,20 +42,38 @@
   {{/if}}
   {{#if (not-eq this.data.stream.videoChannel.provider 'bbb')}}
     <div class="field">
-      <label class="required">{{t 'Video Source URL'}}</label>
+      <label class="required">
+        {{t 'Video Source URL'}}
+        <UiPopup
+          @tagName="i"
+          @class="info icon"
```
Comment (iamareebjamal) [inline app/components/forms/events/view/videoroom-form.hbs:49]:
```suggestion
          @class="info circle icon"
```

File: app/components/forms/events/view/videoroom-form.hbs [hunk 2]
```diff
@@ -42,20 +42,38 @@
...
         @value={{this.data.stream.url}}
         @onChange={{action (mut this.data.stream.url)}} />
     </div>
     <div class="field">
-      <label>{{t 'Room Password'}}</label>
+      <label>
+        {{t 'Room Password'}}
+        <UiPopup
+          @tagName="i"
+          @class="info icon"
```
Comment (iamareebjamal) [inline app/components/forms/events/view/videoroom-form.hbs:62]:
```suggestion
          @class="info circle icon"
```

File: app/components/forms/events/view/videoroom-form.hbs [hunk 3]
```diff
@@ -42,20 +42,38 @@
...
         @value={{this.data.stream.password}} />
     </div>
   {{/if}}
   <div class="field">
-    <label>{{t 'Additional Information'}}</label>
+    <label>
+      {{t 'Additional Information'}}
+      <UiPopup
+          @tagName="i"
+          @class="info icon"
```
Comment (iamareebjamal) [inline app/components/forms/events/view/videoroom-form.hbs:74]:
```suggestion
          @class="info circle icon"
```

File: app/templates/components/tables/default.hbs [hunk 4]
```diff
@@ -34,6 +34,12 @@
             {{else}}
               {{column.name}}
             {{/if}}
+            {{#if column.helperInfo}}
+              <UiPopup
+                @tagName="i"
+                @class="info icon"
```
Comment (iamareebjamal) [inline app/templates/components/tables/default.hbs:40]:
```suggestion
                @class="info circle icon"
```

**Pull request merged**